desk/app/reel.hoon

```diff
@@ -119,16 +119,27 @@
       %reel-describe
     ?>  =(our.bowl src.bowl)
     =+  !<([id=cord =metadata:reel] vase)
-    ?~  (~(has by stable-id) id)  `this
-    ::  the token here is a temporary identifier for the metadata
-    ::  a new one will be assigned by the bait provider and returned to us
-    =/  new-fields  (~(put by fields.metadata) 'bite-type' '2')
+    =/  old-token  (~(get by stable-id) id)
+    =/  new-fields
+      %-  ~(gas by fields.metadata)
+      :~  ['bite-type' '2']
+          ['inviter' (scot %p src.bowl)]
+          ['group' id]
+      ==
     =/  new-metadata  metadata(fields new-fields)
```

Comment:
Any reason not to `=.` here instead of assigning to `new-fields` and `new-metadata`? Or am I missing a place where the original `metadata` gets used?